streams/mx.m3u

```diff
@@ -199,8 +203,10 @@ https://5e50264bd6766.streamlock.net/mexiquense/videomexiquense/playlist.m3u8
 https://stream8.mexiserver.com:1936/misurtv/misurtv/playlist.m3u8
 #EXTINF:-1 tvg-id="MonteMaria.mx",Monte Maria (1080p)
 https://616eee1852d6c.streamlock.net/live/myStream/playlist.m3u8
-#EXTINF:-1 tvg-id="MVMNoticias.mx",MVM Noticias [Geo-blocked]
+#EXTINF:-1 tvg-id="MVSTV.mx",MVS TV [Geo-blocked]
 https://dish.akamaized.net/Content/HLS_HLS_CLR/Live/channel(mvs)/variant.m3u8
+#EXTINF:-1 tvg-id="Nickelodeon.mx",Nickelodeon (480p)
+https://indythenerd.serv00.net/nickelodeonhd.m3u8
```

Comment:
looped.

& not quite sure serv00 will like that x)

Reply:
the source stream is very unstable
i'll remove it for now until i find a stable one